recipes/libgcrypt/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+class LibgcryptConan(ConanFile):
+    name = "libgcrypt"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnupg.org/download/index.html#libgcrypt"
+    description = "Libgcrypt is a general purpose cryptographic library originally based on code from GnuPG"
+    topics = ("conan", "libgcrypt", "gcrypt", "gnupg", "gpg", "crypto", "cryptography")
+    license = "LGPL-2.1-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    requires = 'libgpg-error/1.36'
```

Comment:
Please use a `requirements` method, in the name for conformity and maintainability.